apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/EditBuyAmount/EditBuyAmount.tsx

```diff
@@ -24,10 +26,6 @@ export const EditBuyAmount = () => {
 
   const [inputValue, setInputValue] = React.useState<string>(Quantities.denominated(quantity, tokenInfo.decimals ?? 0))
 
-  React.useEffect(() => {
-    setInputValue(Quantities.denominated(quantity, tokenInfo.decimals ?? 0))
-  }, [quantity, tokenInfo.decimals])
-
```

Comment:
hey @jorbuedo 

this code was originally added in the send flow equivalent component to fix this https://emurgo.atlassian.net/browse/YOMO-655

not sure if you noticed this

if its fixed after you changes perfect!

Reply:
If you know a better solution, even better!

Reply:
I didn't check the send flow. But the useEffect in this case reflected back to the input field the denominated value when quantity changed. With the way I'm doing it now it shouldn't be necessary.

Reply:
El problema si no recuerdo mal estaba en la inicializacion del componente cuando se seleccionaba un nuevo asset. A volver al componente el calor inicial no se actualiza al cambiar el prop (se puede ver en el video del ticket )

Pero si esta arreglado perfecto!!!

Reply:
The problem if I remember correctly was in the initialization of the component when a new asset was selected. To return to the component the initial heat does not update when changing the prop (you can see it in the video of the ticket)

But if its fixed perfect!!!!